mipengine/node/tasks/views.py

```diff
@@ -6,6 +6,8 @@
 from mipengine.node import config as node_config
 from mipengine.node.monetdb_interface.common_actions import create_table_name
 
+DATA_TABLE_PRIMARY_KEY = "row_id"
```

Comment:
In the `csv_importer` this 'row_id' value exists as a hardcoded value.

Let's make this variable a "common" variable in another file and use it from both places. It could go in an init file.